### PR TITLE
Updating rule name(s) and filename(s)

### PR DESCRIPTION
**Pull request by EmmaJP**
Updating rule name(s) and filename(s) ... after discussions on what naming conventions should be.

Closes issue(s):
- #250 

---

### **When creating PR:**

- Make sure you requesting to **pull a branch** (right side) to the `develop` branch (left side).

### **After creating PR:**

- Add yourself (and co-authors) as "Assignees" for PR.
- Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- Close the issue that the PR resolves (and make sure the issue is referenced in the top of this comment)
- Optionally request feedback from anyone in particular by assigning them as "Reviewers".

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/audio-text-alternative-e7aa44.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 id: e7aa44
-name: Audio only has a text alternative
+name: `audio` element content has text alternative
 rule_type: composite
 description: |
-  This rule checks if audio only elements have a text alternative available.
+  This rule checks if `audio` elements have a text alternative available.
```
Comment (audreymaniez) [inline _rules/audio-text-alternative-e7aa44.md]: We should use singular, not plural. And in the previous rule,the "element" is not specified  "This rule checks `audio` is a media alternative for text on the page."

Proposal

>  This rule checks `audio` has a text alternative available

File: _rules/role-required-states-and-properties-4e8ab6.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 id: 4e8ab6
-name: Role has required states and properties
+name: Element with role has required states and properties
```
Comment (audreymaniez) [inline _rules/role-required-states-and-properties-4e8ab6.md]: Backsticks and 'attribute' word no ? 

> Element with *`role` attribute* ?

Comment (carlosapaduarte) [inline _rules/role-required-states-and-properties-4e8ab6.md]: explicit should also be in the name

Comment (EmmaJP) [reply _rules/role-required-states-and-properties-4e8ab6.md]: In this case I believe the role is implicit rather than an attribute

Comment (EmmaJP) [reply _rules/role-required-states-and-properties-4e8ab6.md]: Agreed, though wondering if explicit or implicit

File: _rules/audio-text-alternative-e7aa44.md (hunk 1 above)
Comment (EmmaJP) [reply _rules/audio-text-alternative-e7aa44.md]: I don't recall changing the description yesterday. It shouldn't have been changed.

File: _rules/role-required-states-and-properties-4e8ab6.md (hunk 2 above)
Comment (EmmaJP) [reply _rules/role-required-states-and-properties-4e8ab6.md]: Apparently not ... this rule checks elements with the role attribute.

Comment (EmmaJP) [reply _rules/role-required-states-and-properties-4e8ab6.md]: Explicit ... ie. elements with a role attribute. eg. `<div role="main">

File: _rules/form-control-label-descriptive-cc0f0a.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 id: cc0f0a
-name: Form field label is descriptive
+name: Form control label is descriptive
```
Comment (EmmaJP) [inline _rules/form-control-label-descriptive-cc0f0a.md]: In this situation label is generic and not specifically the `label` element

**Pull request merged**